dotnet/Paymentsense.Connect-e.Api/CardPaymentsService.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Paymentsense.Connect_e.Api.Models;
+using RestSharp;
+
+namespace Paymentsense.Connect_e.Api
+{
+    public class CardPaymentsService : ICardPaymentsService
+    {
+        private readonly CardPaymentsOptions paymentOptions;
+        public CardPaymentsService(IOptions<CardPaymentsOptions> options)
+        {
+            paymentOptions = options.Value;
+        }
+
+        public PaymentTokenResponse? AccessToken(string type)
+        {
+            var token = GetTokenRequest(); 
+            token.transactionType = type;
+
+            var tokenResponse = GetResponse<PaymentTokenResponse>(paymentOptions.ApiUrl + "/access-tokens", Method.Post, token);
+            return tokenResponse;
+        }
+
+        public CrossReferencePaymentResponse? Collection(string id)
```

Comment:
What does the `?` mean in this context. Don't think that syntax existed in my .net days

Reply:
? means nullable, I use it if I want to assign null to value type, I dont use it for ref types as by default it can be null, but in .net 6 or vs2022 you will see warning if you assign null to ref type and not define it as nullable: 
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/nullable-warnings?f1url=%3FappId%3Droslyn%26k%3Dk(CS8603)
so I just decided to do it to resolved all warnings in case this will be enforce some day